_includes/agplv3.html

```diff
@@ -0,0 +1,25 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewbox="0 0 864 358" version="1.1">
```

Comment:
I thought that I had tried this on another Jekyll site and it didn't work, but I just gave it another try and it worked fine, so I've made that change.